### DISCUSSION TITLE
enhancement.  Frigate_motion.py is stale and doesn't compile in .15.   Should be removed or updated.

### DISCUSSION THREAD

**Discussion by bwmcin** [blakeblackshear/frigate#14768] [General Support]
### Checklist

- [X] I have updated to the latest available Frigate version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [incognito mode](https://www.computerworld.com/article/1719851/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) to rule out problems with any third party extensions or plugins I have installed.

### Describe the problem you are having

frigate_motion.py remains in the motion directory.  I use this version vs. the "improved" version as the original version is more sensitive to motion in my setting (I will still try to figure out how to use the improved version).  To make frigate_motion.py to work in .15 several changes were needed.  Since it appears updates are still being made to this file, I would recommend updating this file to be compatible with .15.

### Steps to reproduce

1.Change video.py to import frigate_motion and change the call to improved_motion to frigate_motion.
2.
3.
...

### Version

.15 Beta 1

### Frigate config file

```yaml
Not applicable
```

### docker-compose file or Docker CLI command

```yaml
Not applicable
```

### Relevant Frigate log output

```shell
Not applicable
```

### Relevant go2rtc log output

```shell
Not applicable
```

### Operating system

Other Linux

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

Not applicable

### Screenshots of the Frigate UI's System metrics pages

Not applicable

### Any other information that may be helpful

Diff of original and my file used.  [updated_frigate_motion.txt](https://github.com/user-attachments/files/17613154/updated_frigate_motion.txt)
My file...
[frigate_motion.py.txt](https://github.com/user-attachments/files/17613164/frigate_motion.py.txt)

**Comment by NickM-27** [comment]
this isn't really a bug as it does not affect the way Frigate itself runs. This file has not changed for some time but remains in case we need to use it for anything, most likely just internal debugging. If you want to use the other motion detector then I would suggest making a change on your fork

**Comment by bwmcin** [comment]
No problem, as I mention it as possibly an enhancement, and if you ever plan to use it for internal debugging, the indicated updates will have to be performed.
